command/sidechain/enlisting/params.go

```diff
@@ -0,0 +1,44 @@
+package enlisting
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	sidechainHelper "github.com/0xPolygon/polygon-edge/command/sidechain"
+)
+
+var (
+	newValidatorAddressFlag = "address"
+)
+
+type enlistParams struct {
+	accountDir          string
+	configPath          string
+	jsonRPC             string
+	newValidatorAddress string
+}
+
+func (ep *enlistParams) validateFlags() error {
+	return sidechainHelper.ValidateSecretFlags(ep.accountDir, ep.configPath)
+}
+
+type enlistResult struct {
+	newValidatorAddress string
+}
+
+func (er enlistResult) GetOutput() string {
+	var buffer bytes.Buffer
+
+	var vals []string
```

Comment:
Nitpick: Remove this `var` declaration since you are initializing it in the next line. Just do `vals := make()`

Reply:
or merge together two vars and remove `vals = make([]string, 0, 1)`. `append` is smart enough :)

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1258/commits/8de4d6a725d75f5819e1071cfac7a5ffbab412fb